src/cmd/run.go

```diff
@@ -57,6 +63,22 @@ func doRun(cmd *cobra.Command, args []string) {
 	client.RunnerUnregister(&runner.Id)
 }
 
+func electLeader() {
+	leaseLockName := viper.GetString("runner-deployment")
+	leaseLockNamespace := viper.GetString("runner-pod-namespace")
+	lockIdentity := viper.GetString("runner-pod-name")
+
+	config, err := pkg.GetKubernetesConfig()
+	pkg.CheckErr(err)
```

Comment:
now that this function `electLeader` is being run as a gorouting - this will only exit the go routing - the rest of opslevel-runner will keep running if we fail here and thats bad.  We should move this out to the `doRun` function and pass in the k8s client - that way the whole run will exit instead of being put in a bad state.